front/app/containers/Admin/pagesAndMenu/containers/ContentBuilder/containers/ContentBuilderLanguageProvider.tsx

```diff
@@ -0,0 +1,52 @@
+import React, { useEffect, useState } from 'react';
+
+// Utils
+import { IntlProvider } from 'react-intl';
+import { isNilOrError, NilOrError } from 'utils/helperUtils';
+import moment from 'moment';
+
+// Typings
+import { Locale } from 'typings';
+
+type Props = {
+  contentBuilderLocale: Locale | NilOrError;
+  platformLocale: Locale | NilOrError;
+  children: React.ReactNode;
+};
+
+// Loads correct messages for the contentBuilder locale & change moment locale
+const ContentBuilderLanguageProvider = ({
```

Comment:
Since this is identical to the `ReportLanguageProvider` we could share the code, I don't see this particular component deviating any time soon

Reply:
I'll keep it separate for now but share it between the two once we convert the other report builders to the new system, possibly together with some other pieces of code that can be shared.